coderd/database/migrations/000219_notifications.up.sql

```diff
@@ -0,0 +1,66 @@
+CREATE TYPE notification_message_status AS ENUM (
+    'pending',
+    'leased',
+    'sent',
+    'permanent_failure',
+    'temporary_failure',
+    'unknown'
+    );
+
+CREATE TYPE notification_method AS ENUM (
+    'smtp',
+    'webhook'
+    );
+
+CREATE TABLE notification_templates
+(
+    id             uuid                 NOT NULL,
+    name           text                 NOT NULL,
+    enabled        boolean DEFAULT TRUE NOT NULL,
```

Comment:
Hm.. when exactly can a template be disabled?

Reply:
This is not necessary for now, actually; this will be used in the future when we allow operators to create their own notifications.

I can remove this if you like.

Reply:
I'd say remove it for now.